kivy/lib/osc/OSC.py

```diff
@@ -220,33 +295,36 @@ def parseArgs(args):
 
 
 def decodeOSC(data):
-    """Converts a typetagged OSC message to a Python list."""
-    table = { "i" : readInt, "f" : readFloat, "s" : readString, "b" : readBlob, "d" : readDouble }
-    decoded = []
-    address,  rest = readString(data)
-    typetags = ""
-
-    if address == "#bundle":
-        time, rest = readLong(rest)
-#       decoded.append(address)
-#       decoded.append(time)
-        while len(rest)>0:
-            length, rest = readInt(rest)
-            decoded.append(decodeOSC(rest[:length]))
-            rest = rest[length:]
-
-    elif len(rest) > 0:
-        typetags, rest = readString(rest)
-        decoded.append(address)
-        decoded.append(typetags)
-        if typetags[0] == ",":
-            for tag in typetags[1:]:
-                value, rest = table[tag](rest)
-                decoded.append(value)
-        else:
-            print("Oops, typetag lacks the magic ,")
-
-    return decoded
+    try:
```

Comment:
This one is much more touchy; if incorrect data is coming on TCP connection, the packet should be reported and ignored. If this didn't eat the exception, the program would close with little indication of why.
This should be made more robust. Also; I would like to get 'logging' in here but haven't looked at how much work that'll take yet.